### ISSUE TITLE
Implement accessibility (aria-labels) on every controls by default

### ISSUE THREAD

**Issue by falisca** [JiHong88/SunEditor#1004]
**Describe the bug**
Many buttons in the toolbar are not properly labelled for disabled users, using a ScreenReader for example.
Most notably on each  controls the aria-label attribute, with a correct value, is missing. That causes the ScreenReaders not able to determine what function is associated with a button.
I have tryed to fix this but unfortunately I'm not so expert in js and the source structure of suneditor is too much complicate for me.
Is very inportant to implement correctly accessibility so all users can use SunEditor.
Notice that this problem is present from the demos to the various project that uses SunEditor (e.g: Angular-SunEditor and SunEditor-react)

**To Reproduce**
1. Install and activate a ScreenReader (on Windows NVDA for example, TalkBack or VoiceOver are preinstalled for Android and IOs respectively).
2. Go to the example SunEditor page and use the TAB key or better the B key to search buttons of the toolbar; alternativelì use the up and down arrow keys to navigate the entire page. On mobile use swipes with a finger to left or right.
3. Hear some buttons says what it is (inline button, classic button) and some not (button, button, button).

**Expected behavior**
All ScreenReaders must be told what a button will do if pressed, giving semantical infos using the aria-label attribute properly, see the WAI ARIA rules and specifications.

**Comment by JiHong88** [comment]
The [2.43.6](https://github.com/JiHong88/SunEditor/releases/tag/2.43.6) version has been updated.
If this issue has not been resolved, please reopen this issue.
Thank you.
